### PR TITLE
Reformat /containers/(id)/resize to make it consistent with other sections.

### PR DESCRIPTION
**Pull request by charleswhchan**
Signed-off-by: Charles Chan charleswhchan@users.noreply.github.com

### CODE REVIEW

File: docs/reference/api/docker_remote_api_v1.21.md [hunk 1]
```diff
@@ -690,7 +690,7 @@ Status Codes:
 
 ### Resize a container TTY
 
-`POST /containers/(id)/resize?h=<height>&w=<width>`
+`POST /containers/(id)/resize`
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.21.md]: wondering: are both query parameters *required*? If they are, perhaps that was the reason for including them here (not sure)

File: docs/reference/api/docker_remote_api_v1.21.md [hunk 2]
```diff
@@ -704,6 +704,11 @@ Resize the TTY for container with  `id`. You must restart the container for the
       Content-Length: 0
       Content-Type: text/plain; charset=utf-8
 
+Query Parameters:
+
+-   **h** – height of `tty` session
+-   **w** – width
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.21.md]: I think the "unit" here is number of characters, should we mention that?

File: docs/reference/api/docker_remote_api_v1.21.md (hunk 1 above)
Comment (charleswhchan) [inline docs/reference/api/docker_remote_api_v1.21.md]: Probably, based on the tests here in [docker_api_reize_test.go](https://github.com/docker/docker/blob/e98a8815704c4c4007d45a88f88ab6a45959066e/integration-cli/docker_api_resize_test.go)

Perhaps I should have kept the original format here for /container  (ie. w `h=<height>&w=<width>`) and modify "Exec Resize" instead?
https://docs.docker.com/reference/api/docker_remote_api_v1.19/#exec-resize

File: docs/reference/api/docker_remote_api_v1.21.md (hunk 2 above)
Comment (charleswhchan) [inline docs/reference/api/docker_remote_api_v1.21.md]: Added.

**Comment by SvenDowideit** [comment]
LGTM

File: docs/reference/api/docker_remote_api_v1.21.md (hunk 1 above)
Comment (charleswhchan) [inline docs/reference/api/docker_remote_api_v1.21.md]: @thaJeztah, any further comments on this?

Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.21.md]: Oh, sorry, this slipped my attention. I think it's good like this; the example below shows how to use it, which should explain its use.

File: docs/reference/api/docker_remote_api_v1.21.md [hunk 3]
```diff
@@ -2102,7 +2107,7 @@ Status Codes:
 
 `POST /exec/(id)/resize`
 
-Resizes the `tty` session used by the `exec` command `id`.
+Resizes the `tty` session used by the `exec` command `id`.  The unit is number of characters.
 This API is valid only if `tty` was specified as part of creating and starting the `exec` command.
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.21.md]: Shouldn't the example below include the `?w=xxx&h=xxx` parameters? (Not your fault, but I just noticed this)

Comment (charleswhchan) [inline docs/reference/api/docker_remote_api_v1.21.md]: :smile: You are right. Good catch! 

Fixed.

**Comment by thaJeztah** [comment]
Thanks and LGTM

**Pull request merged**